Parsers/ASimNetworkSession/ProductParsers/ASimNetworkSessionPaloAltoPanOS.yaml

```diff
@@ -0,0 +1,118 @@
+Parser:
+  Title: Palo Alto Network Sessions
+  Version: '0.1'
+  LastUpdated: Dec 20, 2021
+Product:
+  Name: PaloAlto
+Normalization:
+  Schema: NetworkSessions
+  Version: '0.2.0'
+References:
+- Title: ASIM Network Session Schema
+  Link: https://aka.ms/ASimNetworkSessionDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+Description: |
+  This Query Parser maps Palo Alto PanOS Firewall Events (CommonSecurityLogs) to the Advanced SIEM Information Model Network Session schema.
+ParserName: ASimNetworkSessionPaloAltoPanOS
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let NWParser=(disabled:bool=false){
+  CommonSecurityLog | where not(disabled)
+  | where DeviceVendor=="Palo Alto Networks" and (Activity=="TRAFFIC")
+  | parse AdditionalExtensions with "PanOSDstPackets="DstPackets:long";PanOSsrc_packets="SrcPackets:long";start="EventStartTime:datetime";reason="*
+  | project-rename 
+        EventVendor=DeviceVendor // Not documented
+      , EventProduct=DeviceProduct // Not Documented
+      , EventProductVersion=DeviceVersion // Not Documented
+      , DvcHostname=DeviceName   
+      , EventResourceId=_ResourceId  // Not Documented 
+      , NetworkApplicationProtocol=ApplicationProtocol
+      , SrcZone=DeviceCustomString4 
+      , DstZone=DeviceCustomString5 
+      , NetworkRuleName=DeviceCustomString1
+      , NetworkProtocol=Protocol
+      , SrcBytes=SentBytes
+      , DstBytes=ReceivedBytes 
+      , SrcUsername=SourceUserID 
+      , DstUsername=DestinationUserID 
+      , EventSeverity=LogSeverity // not documented
+      , NetworkPackets=DeviceCustomNumber2 
+      , SrcNatIpAddr=SourceTranslatedAddress
+      , DstNatIpAddr=DestinationTranslatedAddress
+      , NetworkConnectionHistory=FlexString1 // Flags
```

Comment:
Do you know what this is?

Reply:
"32-bit field that provides details on session;" https://docs.paloaltonetworks.com/pan-os/8-1/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/traffic-log-fields
This is mapped in NwSessionV1, could be removed

Reply:
WDYT?

Reply:
Map to PaloAltoFflags. In the future we might extract flags from there, for example 0x00800000—session is denied via URL filtering looks interesting, but not a priority.

Reply:
done, (replaced NetworkConnectionHistory)